website/docs/template.html.markdown

```diff
@@ -0,0 +1,116 @@
+---
+layout: "helm"
+page_title: "helm: helm_template"
+sidebar_current: "docs-helm-template"
+description: |-
+
+---
+
+# Data Source: helm_template
+
+Render chart templates locally.
+
+`helm_template` renders chart templates locally and exposes the rendered manifests in the data source attributes. `helm_template` mimics the functionality of the `helm template` command. This data source does not require Tiller. However, any values that would normally be looked up or retrieved in-cluster will be faked locally. Additionally, none of the server-side testing of chart validity (e.g. whether an API is supported) is done.
+
+For further details on the `helm template` command, refer to the [Helm documentation](https://helm.sh/docs/helm/#helm-template).
+
+## Example Usage
+
+### Render all chart templates
+
+The following example renders all templates of the `mariadb` chart of the official Helm stable repository. Concatenated manifests are exposed as output variable `mariadb_instance_manifest`.
+
+```hcl
+data "helm_repository" "stable" {
+  name = "stable"
+  url  = "https://kubernetes-charts.storage.googleapis.com"
+}
+
+data "helm_template" "mariadb_instance" {
+  name       = "mariadb-instance"
+  namespace  = "default"
+  repository = data.helm_repository.stable.metadata.0.name
+
+  chart   = "mariadb"
+  version = "6.8.2"
+
+  set {
+    name  = "service.port"
+    value = "13306"
+  }
+
+  set_sensitive {
+    name = "rootUser.password"
+    value = "s3cr3t!"
+  }
+}
+
+output "mariadb_instance_manifest" {
+  value = data.helm_template.mariadb_instance.rendered
+}
+
+output "mariadb_instance_manifests" {
```

Comment:
`mariadb_instance_manifest` is here twice

Reply:
I have renamed the output `mariadb_instance_manifest` to `mariadb_instance_rendered` to better distinguish it from `mariadb_instance_manifests` in the example.